File: src/backend/internal/domain/project/phase13.go
package project

import "time"

type Phase13RunInput struct {
	BackendBaseURL string `json:"backend_base_url"`
	FrontendURL    string `json:"frontend_url"`
	IncludeDevOps  bool   `json:"include_devops"`
}

type Phase13DeliveryReport struct {
	GeneratedAt   time.Time `json:"generated_at"`
	ProjectID     string    `json:"project_id"`
	ProjectStatus string    `json:"project_status"`
	IncludeDevOps bool      `json:"include_devops"`
	Artifacts     []string  `json:"artifacts"`
	Warnings      []string  `json:"warnings,omitempty"`
}

File: src/frontend/src/types/phase13.ts
import { Phase5CodeFile } from "./phase5";

export interface Phase13RunRequest {
  backend_base_url?: string;
  frontend_url?: string;
  include_devops: boolean;
}

export interface Phase13DeliveryReport {
  generated_at: string;
  project_id: string;
  project_status: string;
  include_devops: boolean;
  artifacts: string[];
  warnings?: string[];
}

export interface DeliveryArtifactGroups {
  documentation: Phase5CodeFile[];
  infrastructure: Phase5CodeFile[];
  summary?: Phase5CodeFile;
}
